eng/common/pipelines/templates/steps/create-pull-request.yml

```diff
@@ -75,19 +76,6 @@ steps:
       -PRTitle "${{ parameters.PRTitle }}"
       -PRBody "${{ coalesce(parameters.PRBody, parameters.CommitMsg, parameters.PRTitle) }}"
       -PRLabels "${{ parameters.PRLabels}}"
-
-- task: PowerShell@2
-  displayName: Tag a Reviewer on PR
-  condition: and(succeeded(), eq(variables['HasChanges'], 'true'))
-  continueOnError: true
-  inputs:
-    pwsh: true
-    workingDirectory: ${{ parameters.WorkingDirectory }}
-    filePath: ${{ parameters.ScriptDirectory }}/add-pullrequest-reviewers.ps1
-    arguments: >
-      -RepoOwner "${{ parameters.RepoOwner }}"
-      -RepoName "$(RepoNameWithoutOwner)"
-      -AuthToken "$(azuresdk-github-pat)"
-      -GitHubUsers "$(${{ parameters.GHReviewersVariable }})"
-      -GitHubTeams "$(${{ parameters.GHTeamReviewersVariable }})"
-      -PRNumber "$(Submitted.PullRequest.Number)"
+      -UserReviewers "$(${{ parameters.GHReviewersVariable }})"
+      -TeamReviewers "$(${{ parameters.GHTeamReviewersVariable }})"
+      -Assignees "$(${{ parameters.Assigness }})"
```

Comment:
We should likely name this similar to the others and call it `GHAssigneesVariable`